app/tasks/datamart/land.py

```diff
@@ -50,8 +52,32 @@ async def compute_tree_cover_loss_by_driver(
         await datamart_crud.update_result(resource_id, resource)
 
     except Exception as e:
+        # To debug this error in AWS Logs Insights:
```

Comment:
I like the simplicity of just logging it well, now we just need to remember actually ever do this... I wonder if there's a way we can set up a dashboard for this? Especially one that can be added in New Relic

Reply:
Yeah, that was my next step. I'll create a card to follow-up after getting this into production and being able to play around with "Logs Insights" and New Relic.